docs/source/interfaces/ISIS_Reflectometry.rst

```diff
@@ -388,22 +387,93 @@ Hovering over the highlighted run with your cursor will allow you to see why the
+| Filter                        | This can be specified to filter out workspaces       |
+|                               | in the workspace list whose name does not match      |
+|                               | that of the filter text.                             |
++-------------------------------+------------------------------------------------------+
+| Regex                         | Checking this option allows a regular expression     |
+|                               | to be used for filtering workspace names.            |
++-------------------------------+------------------------------------------------------+
+| List Of Workspaces            | The left listbox will contain any workspaces loaded  |
+|                               | into mantid. Double clicking on one will fill        |
+|                               | the right list box with the parameters it contains.  |
+|                               | This listbox supports multi-select in order to       |
+|                               | allow for multiple workspaces to be saved out        |
+|                               | at the same time with the same settings.             |
++-------------------------------+------------------------------------------------------+
+| List Of Logged Parameters     | The right listbox starts out empty, but will fill    |
+|                               | with parameter names when a workspace in the left    |
+|                               | listbox is double clicked. This listbox supports     |
+|                               | multi-select in order to allow for the save output   |
+|                               | to contain multiple parameter notes.                 |
++-------------------------------+------------------------------------------------------+
+| Spectra list                  | Spectra list entry allows you to specify what        |
+|                               | spectra you want in the output file. If this is      |
+|                               | blank, all spectra are included.                     |
++-------------------------------+------------------------------------------------------+
+| File format                   | This dialog can save to ANSTO, Ill cosmos, 3-column, |
+|                               | and a customisable format. It doesn't save from      |
+|                               | the main interface's table, but from workspaces      |
+|                               | loaded into mantid. All algorithms are also          |
+|                               | available as save algorithms from mantid itself.     |
++-------------------------------+------------------------------------------------------+
+| Custom Format Options         | When saving in 'Custom' this section allows you      |
+|                               | to specify if you want a Title and/or Q Resolution   |
+|                               | headings as well as specifying the delimiter.        |
```

Comment:
I copied a lot of these from the old ISIS Reflectometry docs. I suppose even they got some things wrong.